### PR TITLE
BACK-1503: Re-assess need for poolconfig in uniswapV3 & forks

### PR DESCRIPTION
**Pull request by aburkut**

### CODE REVIEW

File: src/stateful-event-subscriber.ts [hunk 1]
```diff
@@ -173,6 +181,10 @@ export abstract class StatefulEventSubscriber<State>
     this.isInitialized = true;
   }
 
+  protected getValueToPushIntoNewPoolsChannel() {
+    return {};
```
Comment (mwamedacen) [inline src/stateful-event-subscriber.ts:185]: please make it abstract or throw an error

File: src/constants.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
 
 export const CACHE_PREFIX = 'dl';
 
+export const INIT_SERVICE_CACHE_PREFIX = 'is';
```
Comment (mwamedacen) [inline src/constants.ts:12]: let's stick to dl

Comment (aburkut) [reply src/constants.ts]: updated

File: src/stateful-event-subscriber.ts (hunk 1 above)
Comment (aburkut) [reply src/stateful-event-subscriber.ts:185]: @mwamedacen I didn't get your point here. Why we need to make it abstract? Now getValueToPushIntoNewPoolsChannel method contains default implementation for all pools.

Comment (aburkut) [reply src/stateful-event-subscriber.ts]: updated

**Review by mwamedacen — APPROVED** [review]

**Pull request merged**